server/controllers/finance/reports/balance/index.js

```diff
@@ -34,10 +34,10 @@ function document(req, res, next) {
 
   // date option
   if (params.dateFrom && params.dateTo) {
-    session.dateFrom = new Date(params.dateFrom);
-    session.dateTo = new Date(params.dateTo);
+    session.dateFrom = moment(params.dateFrom).format('YYYY-MM-DD');
```

Comment:
Are you sure this was necessary?  I thought `node-mysql` parsed these for us ... has something changed?